spatialmath/point.go

```diff
@@ -42,6 +42,13 @@ func (pt *point) Label() string {
 	return ""
 }
 
+// SetLabel sets the label of this point.
+func (pt *point) SetLabel(label string) {
+	if pt != nil {
```

Comment:
I dont think this can ever be true

Reply:
Same goes for the other geometry types

Reply:
This code exists because it was causing panics at some point. We can add a TODO to verify that it's no longer needed.

Reply:
I'm confused how this could ever be called though? If you have a nil receiver how does the method even execute?

Reply:
My brain is kinda tripping on this coming from an OO language like C++, but apparently you can call a receiver on an uninitialized pointer as long as the receiver itself is initialized.

https://play.golang.com/p/2pqSWNJ3X5n

Reply:
I see.  we should probably do the correct thing and not be calling methods on initialized pointers, not putting the burden on the Geometry methods though.  I can look into what errors this is causing and try to fix

Reply:
I investigated by removing these guards and ran the unit test suite.  There were no instances where we panicked.